SCRAM/GMake/Makefile.rules

```diff
@@ -19,6 +19,16 @@ TARGETDIR:= $(subst /,_,$(THISDIR))
 .DEFAULT_GOAL := all
 .PHONY: FORCE_TARGET
 ##############################################################################
+ifeq ($(shell [ -t 1 ] >> /proc/$${PPID}/fd/1 && echo terminal || echo pipe),terminal)
```

Comment:
@fwyzard , build and tests look good now. By the way, what is the purpose of `>> /proc/$${PPID}/fd/1 ` ?  
```
ifeq ($(shell [ -t 1 ] && echo terminal || echo pipe),terminal)
```
should work too ... right?

Reply:
ignore my comment, yes looks like your change does the rigth thing :-)

Reply:
The trick is to make the shell use the same stdout as the parent process, that is `make` itself.
Otherwise the check applies to the process spawned by `make`, which is always attached to a pipe and not to a terminal.